docs/guide/audit-trail.md

```diff
@@ -0,0 +1,71 @@
+
+- Adding or removing account members
+- Changing roles of account members
+- Adding or removing App guests
+- Changing roles of App guests
+- Conversion of the account to a team account
+
+### App Lifecycle and Auto Monitoring
+
+Tracks important changes related to your [Apps](https://sematext.com/docs/guide/app-guide/) and their monitoring setup. It includes the creation, disabling, and deletion of [Apps](https://sematext.com/docs/guide/app-guide/) across all Sematext solutions. It also covers special cases like Apps disabled due to failed payments. Additionally, it tracks when automatic metrics or logs monitoring and shipping are enabled or disabled, along with changes to the destination App for logs or metrics.
+
+### Alert Rule Changes
+
+Tracks when alerts are created, deleted, enabled, or disabled, as well as any updates made to [alert rules](https://sematext.com/docs/guide/alerts-guide/). This includes changes to alert conditions, queries, filters, schedules, names, priorities, group-by and roll-up settings, metrics, and transformation scripts. It also captures changes to notification settings and hooks.
+
+### Synthetics Monitoring
+
+Tracks creation, updates, disabling, and deletion of [monitors](https://sematext.com/docs/synthetics/). This includes:
+
+- Changes to monitor URLs, device types, run intervals, [locations](https://sematext.com/docs/synthetics/#locations), and conditions
+- [Status pages](https://sematext.com/docs/synthetics/status-pages/) and [scheduled pauses](https://sematext.com/docs/synthetics/scheduled-pauses/) being created, updated, or deleted
+- [SSL certificate tracking](https://sematext.com/docs/synthetics/ssl-certificate-monitoring/) and validation toggled on or off
+- Updates to request settings and authentication
+- Monitor type changes (URL/script-based), as well as script and secret variable updates
+
+### Pipelines and Processors
+
+Records creation, updates, removal, enabling, and disabling of [processors](https://sematext.com/docs/logs/processors-overview/). Tracks detailed changes to processor settings such as field mappings, filters, target/source fields, scripts, sampling rates, and processor ordering.
+
+###  Reports and Dashboards
+
+Tracks creation and updates of reports at both App and Dashboard levels, updates to [report components](https://sematext.com/docs/dashboards/reports-and-components/), and creation or deletion of [dashboards](https://sematext.com/docs/dashboards/).
+
+> When multiple settings are changed at once—for example, if several fields in an alert rule are edited—Audit Trail groups them into a single event. Each recorded entry shows what was changed, along with the previous and new values for each field.
+
+![Audit Trail Flyout](/docs/images/guide/audit-trail/audit-trail-event-flyout.png)
+
+## How to Use It
+
+You can access the Audit Trail from your Sematext Cloud account. It allows you to:
+
+- Filter events by user, event type, or related App
```

Comment:
*related* App?

Reply:
Updated to:
Filter events by user, event type, or by the App where the user performed the action